Directory.Build.props

```diff
@@ -286,6 +315,7 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Deterministic>false</Deterministic>
+    <CheckForOverflowUnderflow>false</CheckForOverflowUnderflow>
```

Comment:
Can you add a comment here?  How bad was it when this was set to false?  Could we instead move this just to the individual projects that need it?

Reply:
It was pretty bad there were a large set of tests in lots of projects failing because this was turned on by default in arcade sdk. We have made attempts to enable it on some projects in the past but we decided against it.

Reply:
Could we turn it on for just src projects?

Reply:
Unfortunately not there are a lot of issues throughout and libraries that want to enable it should do it for just there projects.